source/examples.rst

```diff
@@ -9,4 +9,12 @@ Text to image GUI
 
 See : `EMF Demos - Text to Image GUI <https://github.com/easy-model-fusion/demos/tree/main/text-to-image-gui>`_
 
-The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
\ No newline at end of file
+The code defines a graphical user interface (GUI) application using the Tkinter library. This application allows users to generate images from text using the `Stabilityai - Stable Diffusion XL Base 1.0 <https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0>`_ model.
+
+
+Images from Text Prompts Generated by Text Model GUI
+----------------------------------
```

Comment:
have you compiled locally and checked ?
i am not quite sure but if i remember correctly you might encounter an error as the title's text is wider than the line underneath it